packages/browser-wallet-api/README.md

```diff
@@ -2,80 +2,7 @@
 
 This package is an implementation of an API to be used in web applications for communicating with the Concordium browser wallet.
 
-## Using the API
```

Comment:
Hmm... I don't think I agree on deleting this actually, but we should also avoid duplication. Maybe we should reference the types package from here, or reference the API package from the types package? Leaning most towards having the full document on the package published on NPM.

Reply:
Yes, I agree, that's why I link to the types package below this.